src/standalone/chat/installPackageTool.ts

```diff
@@ -60,16 +61,33 @@ export class InstallPackagesTool implements vscode.LanguageModelTool<IInstallPac
             success = await installPackageThroughEnvsExtension(kernelUri, packageList);
         }
 
-        if (!success) {
-            // TODO: There is an IInstaller service available, but currently only for a set list of packages.
-            const result = await sendPipInstallRequest(kernel, packageList, token);
-            output = ` Output: ${String(result)}`;
-            if (output.length > 200) {
-                output = output.substring(0, 200) + '...';
+        if (!success && kernel.kernelConnectionMetadata.interpreter) {
```

Comment:
@amunger  perhaps your didn't realize this, now this will no longer work with remote kernels.